freshsales-suite/contact-kyc/config/iparams.json

```diff
@@ -0,0 +1,14 @@
+{
+    "fcrm_domain": {
+        "display_name": "Freshsales Domain",
+        "description": "Please enter your Freshsales domain web address",
+        "type": "text",
```

Comment:
Use either `"type": "domain"` or `"type": "url"`.

Reply:
Have used `"type":"url"` as this is an omni app would have to handle for both *Freshsales* and *Freshworks_crm*.

Reply:
The `fcrm_api_key` is retained as text as again if we change the type to `api_key`, it needs to handle an associated product and in this case it's an omniapp.